src/server/game/Entities/Player/Player.h

```diff
@@ -404,7 +404,7 @@ enum PlayerFlags
     PLAYER_FLAGS_CONTESTED_PVP          = 0x00000100,       // Player has been involved in a PvP combat and will be attacked by contested guards
     PLAYER_FLAGS_IN_PVP                 = 0x00000200,
     PLAYER_FLAGS_HIDE_HELM              = 0x00000400,
-    PLAYER_FLAGS_HIDE_CLOAK             = 0x00000800,
+    PLAYER_FLAGS_WAR_MODE               = 0x00000800,
```

Comment:
so, which one is it

Reply:
In my sniff it seem that 0x800 is re-used for warmode (hide_cloak & hide_helm are unused since this is now part of the transmog system)